_returners/splunk_nebula_return.py

```diff
@@ -308,17 +304,17 @@ def flushBatch(self):
 
         if len(self.batchEvents) > 0:
             headers = {'Authorization': 'Splunk ' + self.token}
-            self.server_uri = [x for x in self.server_uri if x[1] is not "bad"]
+            self.server_uri = [x for x in self.server_uri if x[1] is not False]
             for server in self.server_uri:
                 try:
                     r = requests.post(server[0], data=' '.join(self.batchEvents), headers=headers, verify=http_event_collector_SSL_verify, proxies=self.proxy, timeout=self.timeout)
                     r.raise_for_status()
-                    server[1] = "good"
+                    server[1] = True
                     break
                 except requests.exceptions.RequestException:
-                    log.error('Request to splunk server "%s" failed. Marking as bad.' % server[0])
-                    server[1] = "bad"
+                    log.info('Request to splunk server "%s" failed. Marking as bad.' % server[0])
+                    server[1] = False
                 except Exception as e:
-                    log.error('Request to splunk threw an error: {0}'.format(e))
+                    log.info('Request to splunk threw an error: {0}'.format(e))
```

Comment:
Oh, I didn't realize what section we're in. Yeah, you'd most certainly want to see that.

IMO, you might also want to fire an event, so something shows up at master even if you somehow managed to set your returners to splunk-only.